test/test-execute/exec-systemcallfilter-system-user.service

```diff
@@ -2,7 +2,7 @@
 Description=Test for SystemCallFilter in system mode with User set
 
 [Service]
-ExecStart=/bin/echo "Foo bar"
+ExecStart=/bin/sh -c 'echo "Foo bar"'
```

Comment:
I don't think that's a change for the better. I don't think anything is wrong with calling /bin/echo and such.

Reply:
... the location of those binaries is pretty much fixed. Distributions with /usr-merge don't care at all, so it's just a matter of finding the path that works on common split-/usr distros.

Reply:
Using `/bin/sh -c` is an easy way to get everything working. I'm not sure why anyone has to waste their time figuring out that `/usr/bin/test` should be used instead of `/bin/test` and `/bin/uname` should be used instead of `/usr/bin/uname`.

Reply:
OK, since you both think that's better, I won't argue.